app/homepage/index.html

```diff
@@ -29,9 +29,18 @@ <h2>You don't have javascript enabled.</h2>
             <a id="rover-map-link" href="/user/rover">Rover</a>
             <a id="calculator-link" href="/user/calculator">Calculator</a>
             <a id="account-link" href="/user/account">Account</a>
-            <a id="logout-link" href="/user/logout">Logout</a>
+            <a id="logout-link" href="">Logout</a>
         </nav>
         <h1>Mars Rover Project</h1>
+        <div id="logout-modal">
+            <div id="modal-content">
+                <h3>Are you sure you would like to logout?</h3>
+                <div class="logout-buttons">
+                    <a href="/user/logout">Ok</a>
```

Comment:
A bit of UX, I'd be slightly confused if it said "Ok" when asked a yes/no question.

Maybe they should be "Logout" and "Cancel"?

Reply:
Done